docs/configuring-metabase/config-file.md

```diff
@@ -97,4 +97,29 @@ config:
       details: ...
 ```
 
+## Users and Admins
+
+A user will be Admin of the instance if it's the first user created. Otherwise you can specify a user as an admin by using the `is_superuser` key.
```

Comment:
```suggestion
The first user created in a Metabase instance is an Admin. The first user listed in the config file may be designated an admin, but not necessarily, for example if someone has already spun up and logged into that Metabase for the first time, that user would be the automatically designated admin. Additionally, you can specify a user account as an admin by using the `is_superuser: true` key.
```

@paoliniluis I'm trying to elaborate here for clarity, as I understood it. Is the above still true?

Reply:
exactly! thanks for the clarity!